Web/Assignment-4/FormJquery.js

```diff
@@ -0,0 +1,50 @@
+//To display data in empty div
+$(document).ready(function() {
+	$("#name").keyup(function() {
```

Comment:
ACTION: Instead of writing keyup event for each element, you should provide a common css class for all elements in html and create a single keyup event for this.